db-migrations/8_add_survey_mode_column.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE survey.survey ADD COLUMN IF NOT EXISTS surveymode character varying(20) NOT NULL;
+
+INSERT INTO survey.survey(surveymode) VALUES('eQ') WHERE shortname IN ('RSI', 'MWSS', 'QBS','MBS');
```

Comment:
https://github.com/ONSdigital/response-operations-ui/pull/533/files#diff-0a48312598377b5d8ffdd031f6bf28f0R76 for a more accurate list of EQ surveys